test/imaginary.js

```diff
@@ -40,7 +40,7 @@ suite('Imaginary', function () {
 
   test('#crop', function (done) {
     Imaginary('./test/fixtures/test.jpg')
-      .server('http://localhost:8088')
+      .server('http://localhost:8088', { timeout: 5000 })
```

Comment:
This is fine, but we could also add method for this to be more expressive and take adventadge of the fluent API design. E.g: client.options({}) or client.httpParams({}).